lxd/device/nic_bridged.go

```diff
@@ -155,6 +159,10 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 				return errors.Wrapf(err, "Invalid network ipv6.address")
 			}
 
+			if d.config["ipv4.address"] == "none" && !shared.IsTrue(d.config["security.ipv4_filtering"]) {
```

Comment:
Shouldnt this be for IPv6, as you handled IPv4 above?

Reply:
Yes. Sorry, should have looked closer there. These should be fixed now.